.tekton/windows-machine-config-operator-bundle-release-4-15-pull-request.yaml

```diff
@@ -10,8 +10,7 @@ metadata:
     pipelinesascode.tekton.dev/on-cel-expression: |
       event == "pull_request"
       && target_branch == "release-4.15"
-      && !files.all.all(x, x.matches('^docs/|\\.md$|^(?:.*/)?(?:\\.gitignore|OWNERS|PROJECT|LICENSE)$'))
-  creationTimestamp: null
+      && "Containerfile.bundle".pathChanged()  && ( "Containerfile.bundle".pathChanged() || "bundle/***".pathChanged() )
```

Comment:
This is different from 4.18 
https://github.com/openshift/windows-machine-config-operator/blob/release-4.18/.tekton/windows-machine-config-operator-bundle-release-4-18-pull-request.yaml#L13